pengwin-setup.d/explorer.sh

```diff
@@ -27,8 +27,15 @@ Windows Registry Editor Version 5.00
 @="_PengwinPath_ run \\"login_shell \\\\\\"%V\\\\\\"\\""
 EOF
 
+      local icopath_win="$(wslpath "$(cmd-exe /C "echo %USERPROFILE%\\pengwin" | tr -d '\r')")"
+      if [ ! -d $icopath_win ]; then
+        mkdir -p $icopath_win
+      fi
+
+      cp /usr/local/lib/pengwin.ico "$icopath_win"
+
       local fullexec=$(wslpath -m "$(which ${exec_name})" | sed 's$/$\\\\\\\\$g')
-      local icopath=$(cmd-exe /C "echo '%USERPROFILE%\\AppData\\Local\\Packages\\WhitewaterFoundryLtd.Co.16571368D6CFF_kd1vv0z0vy70w\\LocalState\\rootfs\\usr\\local\\lib\\pengwin.ico'" | tr -d '\r' | sed 's$\\$\\\\\\\\$g')
+      local icopath=$(cmd-exe /C "echo '%USERPROFILE%\\pengwin\\pengwin.ico'" | tr -d '\r' | sed 's$\\$\\\\\\\\$g')
```

Comment:
Use Pengwin in camel case to be compatible with https://github.com/WhitewaterFoundry/pengwin-setup/blob/613a415304a189a4f6ddd303fc7c738ec19ba703/pengwin-setup.d/home-backup.sh#L5 even though Windows is case insensitive the directory will be created with the case specified.

Reply:
Thanks for the notice